rhwp/rhwp-studio/src/ui/agent-sidebar/agent-sidebar.css

```
@@ -5397,3 +5397,475 @@ html.desktop-mac .ag-fullscreen .ag-environment-panel {
+.ag-fleet-sum,
+.ag-fleet-clock {
+  min-width: 34px;
+  text-align: right;
+}
+
+.ag-fleet-chevron {
+  position: absolute;
+  right: 10px;
+  width: 12px;
+  height: 12px;
+  color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)